### PR TITLE
Add keycloak token

### PR DESCRIPTION
**Pull request by abhijeetnarvekar**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -206,6 +206,26 @@ export interface InitParameters extends SidebarConfiguration {
    * New since sidebar version 14.11.0.
    */
   accessToken?: string;
+
+  /**
+   * KeyClaok tokens used used by sidebar to authenticate in Acrolinx One
```
Comment (bluesun) [inline src/index.ts:211]: Typo in KeyClaok

Comment (abhijeetnarvekar) [reply src/index.ts]: Fixed

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@acrolinx/sidebar-interface",
   "description": "The interface of the Acrolinx Sidebar",
-  "version": "15.3.1",
+  "version": "15.3.2",
```
Comment (JustinHorn) [inline package.json:4]: I don't want to be religious, but I guess it should rather be a new feature version then a patch version

**Review by JustinHorn — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (abhijeetnarvekar) [reply package.json]: Updated to 15.4.0

**Review by JustinHorn — APPROVED** [review]

**Pull request merged**